.env

```diff
@@ -1 +1,3 @@
-REACT_APP_BAND_BUDDY_API_URL=http://localhost:4000
\ No newline at end of file
+REACT_APP_BAND_BUDDY_API_URL=http://localhost:3001
```

Comment:
please delete this whole file. We don't want .env in github

Reply:
done